components/meter/index.css

```diff
@@ -15,15 +15,19 @@
 .spectrum-Meter {
 	--mod-progressbar-max-size: var(--mod-meter-max-width, var(--spectrum-meter-maximum-width));
 	--mod-progressbar-min-size: var(--mod-meter-min-width, var(--spectrum-meter-minimum-width));
+	--spectrum-meter-help-text-to-progress-bar: var(--spectrum-spacing-75);
+	--mod-progressbar-thickness: var(--spectrum-meter-thickness-medium);
```

Comment:
@aramos-adobe thanks for adding the meter tokens in! You're right that they're the same as progress bar. When I made that comment yesterday, I was looking through the large/mobile values 🤦‍♀️ , which are, in fact, different. But either way, I think it's a good practice for us to be explicit with the tokens like this (even if they're the same for the time being). Thanks!